data/core/units/humans/Loyalist_Halberdier.cfg

```diff
@@ -15,7 +15,7 @@
     {AMLA_DEFAULT}
     cost=52
     usage=fighter
-    description= _ "A halberd is a heavy weapon and difficult to use, but very powerful in the hands of an expert. It is also much more flexible than the spear from which it descended. As any halberdier can tell you, the weapon possesses four primary striking points, two more than a spear or pike; these are the tip, the blade, the spike at the base of the shaft, and the inner point on the blade, which can be used in a motion pulling back towards the wielder. All this makes for a very versatile weapon in melee, especially against cavalry.
+    description= _ "A halberd is a heavy weapon and difficult to use, but very powerful in the hands of an expert. It is also much more flexible than the spear from which it descended. As any Halberdier can tell you, the weapon possesses four primary striking points, two more than a spear or pike; these are the tip, the blade, the spike at the base of the shaft, and the inner point on the blade, which can be used in a motion pulling back towards the wielder. All this makes for a very versatile weapon in melee, especially against cavalry.
 
 However, a halberd is notably more expensive to craft than a spear, and in the hands of a novice, much less effective. In the armies of Wesnoth, it is customary only to grant these weapons to veteran pikemen, who have proven that they have the skill to employ them properly on the field of war."
```

Comment:
If 'verteran pikemen' is referring to the Halberdier's previous form, should capitalise pikemen too.